data/scripts/field_move_scripts.inc

```diff
@@ -1,3 +1,8 @@
+.macro IsFollowerFieldMoveUser var:req
+callfunc ScrFunc_IsFollowerFieldMoveUser
```

Comment:
Reiterating my personal preference to just get rid of `callfunc`

Reply:
Let me see what's needed to have expansion's callnative be the replacement for callfunc